### PR TITLE
[cloud_security_posture] Ensure event.kind is correctly set for pipeline errors

### PR DESCRIPTION
**Pull request by MakoWish**
## What does this PR do?

Modify cloud_security_posture integration to correctly set `event.kind` for pipeline errors and ensure `error.message` is an array.

## Checklist

- [X] I have reviewed tips for building integrations and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [X] I have added an entry to my package's changelog.yml file.
- [x] I have verified that Kibana version constraints are current according to guidelines.
 
## Author's Checklist

- [x]  

## Related issues

* For #6582

## Screenshots

### CODE REVIEW

File: packages/cloud_security_posture/manifest.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 format_version: 2.3.0
 name: cloud_security_posture
 title: "Security Posture Management"
-version: "1.5.0-preview23"
+version: "1.6.0-preview24"
```
Comment (oren-zohar) [inline packages/cloud_security_posture/manifest.yml:4]: if we want to include this change in 8.10
```suggestion
version: "1.5.0-preview24"
```

File: packages/cloud_security_posture/changelog.yml [hunk 2]
```diff
@@ -5,6 +5,11 @@
 # 1.3.x - 8.8.x
 # 1.2.x - 8.7.x
 
+- version: "1.6.0-preview24"
```
Comment (oren-zohar) [inline packages/cloud_security_posture/changelog.yml:8]: if we want to include this change in 8.10
```suggestion
- version: "1.5.0-preview24"
```

**Review by oren-zohar — CHANGES_REQUESTED** [review]
Thank you :pray:

**Review by oren-zohar — APPROVED** [review]

**Comment by MakoWish** [comment]
@oren-zohar, can you `/test` again? I don't have permissions to.

**Comment by oren-zohar** [comment]
`/test`

**Pull request merged**